plugins/goodix-fp/goodixfp.quirk

```diff
@@ -0,0 +1,48 @@
+# Goodix Fingerprint sensor
+[DeviceInstanceId=USB\VID_27C6&PID_60A2]
+Plugin = goodixfp
+Flags = self-recovery,dual-image,updatable,use-runtime-version
```

Comment:
if all devices have a flag, maybe just set in `fu_goodixfp_device_init` with `fu_device_add_flag` -- it saves a lot of string parsing.